shared-data/js/__tests__/labwareDefSchemaV3.test.ts

```diff
@@ -113,10 +118,17 @@ describe(`test labware definitions with schema v3`, () => {
     const labwareDef = require(labwarePath) as LabwareDefinition3
 
     test('extents properly use back-left bottom origin with quadrant IV coordinates', () => {
```

Comment:
Just a test name update:

```suggestion
    test('slotFootprintAsChild properly uses back-left bottom origin with quadrant IV coordinates', () => {
```